tox.ini

```diff
@@ -7,7 +7,7 @@ commands = bash .ci/flake8_wrapper.sh
 whitelist_externals = bash
 deps =
     flake8
-    flake8-docstrings==1.0.2
+    flake8-docstrings
```

Comment:
I still don't understand what is the logic behind this pinning unpinning. It seems that not only we pinned/unpinned for 17.01/dev but we also changed dev in the process.